TSC_SUMMARY.md

```diff
@@ -9,6 +9,60 @@ The Technical Steering Committee is a group comprised of people from companies a
 - Omry Hay ([@omry-hay](https://github.com/omry-hay)) representing env0
 - Roni Frantchi ([@roni-frantchi](https://github.com/roni-frantchi)) temporarily filling in for Omry Hay at env0
 
+
+## 2024-02-17
+
+### Attendees
+
+- Igor Savchenko ([@DiscyDel](https://github.com/DicsyDel))
+- Marcin Wyszynski ([@marcinwyszynski](https://github.com/marcinwyszynski))
+- Roger Simms ([@allofthesepeople](https://github.com/allofthesepeople))
+- Roni Frantchi ([@roni-frantchi](https://github.com/roni-frantchi))
+- Wojciech Barczyński([wojciech12](https://github.com/wojciech12))
+
+### Absent
+
+- Omry Hay ([@omry-hay](https://github.com/omry-hay))
+- Yevgeniy Brikman ([@brikis98](https://github.com/brikis98))
+
+### Agenda
+
+1. Decide on producing more specific migration paths from Terraform to OpenT
```

Comment:
A nitpick, but please change `OpenToFu` to `OpenTofu`. Also relevant to lines 38 and 64.